<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alembic Batch Importer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_banner">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../../WEEK_08/ui_assignment/abc_batch_ui.qrc">:/banner/ui_banner.jpg</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QCheckBox" name="chbx_full_path">
        <property name="toolTip">
         <string>Enter your material tagging attribute name.</string>
        </property>
        <property name="text">
         <string>View Full Path</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item alignment="Qt::AlignLeft">
       <widget class="QListWidget" name="listWidget">
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="btn_add_abc">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Add Alembic Files</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_exc_abc">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Remove Alembic Files</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QCheckBox" name="chbx_mat_ovr">
            <property name="text">
             <string>Custom Material Attr:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="l_edit_mat_ovr">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>mTag</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_import">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Import</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_mat_context">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Jump to Materials</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_info">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Info</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cancel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../WEEK_08/ui_assignment/abc_batch_ui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>chbx_mat_ovr</sender>
   <signal>toggled(bool)</signal>
   <receiver>l_edit_mat_ovr</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
